doc/source/serve/production-guide/monitoring.md

```diff
@@ -307,8 +307,32 @@ The following metrics are exposed by Ray Serve:
    * - ``serve_http_request_latency_ms`` [*]
      - * route
        * application
-
      - The end-to-end latency of HTTP requests (measured from the Serve HTTP proxy).
+   * - ``serve_multiplexed_model_load_latency_s``
+     - * deployment
+       * replica
+       * application
+     - The time it takes to load a model.
+   * - ``serve_multiplexed_model_unload_latency_s``
+     - * deployment
+       * replica
+       * application
+     - The time it takes to unload a model..
+   * - ``serve_num_multiplexed_models``
+     - * deployment
+       * replica
+       * application
+     - The number of models loaded on the current replica.
+   * - ``serve_multiplexed_models_unload_counter``
+     - * deployment
+       * replica
+       * application
+     - The counter for unloaded models on the current replica.
+   * - ``serve_multiplexed_models_load_counter``
+     - * deployment
+       * replica
+       * application
+     - The counter for loaded models on the current replica.
```

Comment:
serve_num_multiplexed_models mean how many models you have inside the replica, it can equals to the serve_multiplexed_models_load_counter before it hit the max_num_models_per_replica.

Reply:
The `serve_multiplexed_models_unload_counter` and `serve_multiplexed_models_load_counter` are not for per model stats, it is used for total number of counters.
rephrased to 
`The number of times models unloaded on the current replica.`